### ISSUE TITLE
Wrong locations with line endings LF on windows.

### ISSUE THREAD

**Issue by leitwolf** [oxyc/luaparse#58]
code:
```
--[[


THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
THE SOFTWARE.

]]

local json = {}
 
```
when with line endings LF (the correct line 14, not 12):
```{
    "type": "Chunk",
    "body": [
        {
            "type": "LocalStatement",
            "variables": [
                {
                    "type": "Identifier",
                    "name": "json",
                    "loc": {
                        "start": {
                            "line": 12,
                            "column": 6
                        },
                        "end": {
                            "line": 12,
                            "column": 10
                        }
                    },
                    "isLocal": true
                }
            ],
            "init": [
                {
                    "type": "TableConstructorExpression",
                    "fields": [],
                    "loc": {
                        "start": {
                            "line": 12,
                            "column": 13
                        },
                        "end": {
                            "line": 12,
                            "column": 15
                        }
                    }
                }
            ],
            "loc": {
                "start": {
                    "line": 12,
                    "column": 0
                },
                "end": {
                    "line": 12,
                    "column": 15
                }
            }
        }
    ],
    "loc": {
        "start": {
            "line": 12,
            "column": 0
        },
        "end": {
            "line": 12,
            "column": 15
        }
    },
    "globals": []
}
```
correct with line endings CRLF.

system:
windows10 1803

**Comment by fstirlitz** [comment]
Fixed in 92553684921699174705f10ab0afbad8975accfd, thank you.

**Comment by leitwolf** [comment]
npm still version 0.2.1.
would you please push the new version to npm?
thank you!

**Comment by fstirlitz** [comment]
I think it's something only @oxyc can do. And we haven't actually made a decision whether to make a release yet. A couple of issues are left to be resolved before that happens, like #33 and #54. So don't hold your breath. Sorry.
